torchtune/modules/peft/peft_utils.py

```diff
@@ -340,3 +365,14 @@ def validate_missing_and_unexpected_for_lora(
                 raise AssertionError(f"Missing LoRA key {k} from adapter state dict")
     if lora_unexpected:
         raise AssertionError("Unexpected key loading adapter")
+
+
+def load_dora_magnitudes(model: nn.Module) -> None:
```

Comment:
Fyi after https://github.com/pytorch/pytorch/pull/132954/files landed we can now remove this hack by following the example there